packages/cli/src/zosjobs/submit/data-set/DataSet.definition.ts

```diff
@@ -67,6 +67,15 @@ export const DataSetDefinition: ICommandDefinition = {
             description: "A file extension to save the job output with. Default is '.txt'.",
             implies: ["directory"],
             type: "string"
+        },
+        {
+            name: "jcl-symbols", aliases: ["js"],
+            description: "A string of JCL symbols to use for substitution. " +
```

Comment:
It would be ideal to have a file for the "zosjobs" package where shared string constants are defined, rather than needing to duplicate them in each command definition. The "zosfiles" package (`packages/cli/src/zosfiles/-strings-/en.ts`) has such a file, but since "zosjobs" doesn't, this is just a remark, not a request for changes 🙂

Reply:
I thought about that when adding the --jcl-symbols definition. Because that was inconsistent with all other definitions in this component, I keep the duplicated text in-line.

On this same subject, because the --jcl-symbols description contains a few lines, it tripped a SonarCube detection for duplicated lines of code.

Do you think it would be more (or less) confusing to move just the definition of --jcl-symbols to a common file?
I feel like reworking all of the job submit flags would be beyond the scope of this PR.

Reply:
> Do you think it would be more (or less) confusing to move just the definition of --jcl-symbols to a common file?
I feel like reworking all of the job submit flags would be beyond the scope of this PR.

I agree that reworking all flags seems outside of scope. I'll leave it up to you whether or not to move the definition of `--jcl-symbols` to a separate file. On one hand it would be a bit odd to have it by itself in a separate file. On the other hand, I don't think separating it would cause confusion, and if we want to have a common file we need to start somewhere. 🙂

Reply:
Moving just --jcl-symbols into strings would not provide a very good framework for future storage of other zos-job strings. Doing all of zos-jobs was beyond the scope of this PR. Moving just the SUBMIT definitions seemed like the right compromise.